### ISSUE TITLE
provide a link to BaiduDrive

### ISSUE THREAD

**Issue by Sabermu** [ChrisDud0257/HGGT#3]
You have done a good job. If it's convenient, I would like to provide a link to BaiduDrive with data sets, models, and results

**Comment by ChrisDud0257** [comment]
I am very sorry. Since I am not in the Chinese Mainland now, even if I become a SVIP membership for BaiduDrive, it will still limit my uploading speed to under 100 kb/s. So I just give up then.

**Comment by Sabermu** [comment]
Thank you very much for your reply. I also have a question about the paper. I hope the author can help me answer it when you has time. In the text, 3.2 Design of the Enhancement Models, the first paragraph mentions that the size of the input LR image is consistent with HR. Is it directly adjusted using bicubic downsampling? If my question bothers you, I'm deeply sorry.

**Comment by ChrisDud0257** [comment]
No. The detailed degradations factors could be found in the supplementary. The original paper has explained it in Section 3.2.

> Thank you very much for your reply. I also have a question about the paper. I hope the author can help me answer it when you has time. In the text, 3.2 Design of the Enhancement Models, the first paragraph mentions that the size of the input LR image is consistent with HR. Is it directly adjusted using bicubic downsampling? If my question bothers you, I'm deeply sorry.
